zerver/lib/actions.py

```diff
@@ -1372,8 +1375,8 @@ def check_message(sender, client, message_type_name, message_to,
             'sender_queue_id': sender_queue_id, 'realm': realm}
 
 def internal_prep_message(realm, sender_email, recipient_type_name, recipients,
-                          subject, content):
-    # type: (Realm, Text, str, Text, Text, Text) -> Optional[Dict[str, Any]]
+                          subject, content, send_internal_notice=False):
+    # type: (Realm, Text, str, Text, Text, Text, Optional[bool]) -> Optional[Dict[str, Any]]
```

Comment:
this is a `bool`, not an `Optional[bool]`.  Same with `check_message` and other changes in this file.  Optional = "could be None", not "is a kwarg".